### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by mrantry**
Adding in some style updates. Using Material-UI's `<Typography />` component allows for consistent styles across the board.

Also, set up `package.json`  for testing. Jest will look in the test folder for any file named `*ComponentName*Spec`. For example, a `<Foo />` component test would look like `FooSpec.js`

### CODE REVIEW

**Comment by mrantry** [comment]
Oh, also fixed a vulnerability that looked like it was coming from the boilerplate code. Used `npm audit fix`, so may want to double check that everything is still working properly. I may start working on getting full jest coverage for this, but I ran into some issues with enzyme's `createShallow`

File: package.json [hunk 1]
```diff
@@ -7,20 +7,21 @@
     "@material-ui/icons": "^3.0.1",
     "axios": "^0.18.0",
     "classnames": "^2.2.6",
+    "jest": "^23.6.0",
```
Comment (accimeesterlin) [inline package.json]: No need to install jest, it's part of the `create-react-app` ecosystem.

File: package.json [hunk 2]
```diff
@@ -7,20 +7,21 @@
...
     "lodash": "^4.17.11",
     "moment": "^2.23.0",
     "node-sass": "^4.10.0",
     "prop-types": "^15.6.2",
-    "react": "^16.6.3",
+    "react": "^16.7.0",
```
Comment (accimeesterlin) [inline package.json]: Please revert back to `16.6.3`. If you'd like to upgrade, I'd recommend doing that into a separate PR.

File: package.json [hunk 3]
```diff
@@ -7,20 +7,21 @@
...
     "react-dom": "^16.6.3",
     "react-router-dom": "^4.3.1",
-    "react-scripts": "2.1.1",
+    "react-scripts": "^2.1.3",
```
Comment (accimeesterlin) [inline package.json]: Revert this to `2.1.1` as well

File: package.json [hunk 4]
```diff
@@ -7,20 +7,21 @@
...
     "stringquery": "^1.0.8"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "NODE_ENV=test jest",
```
Comment (accimeesterlin) [inline package.json]: No need for this, the `create-react-app` utils already have jest integrated for us.

File: package.json [hunk 5]
```diff
@@ -31,5 +32,11 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "jest": {
```
Comment (accimeesterlin) [inline package.json]: No need for this too as well

**Pull request closed without merging**